app/main.py

```diff
@@ -1,2 +1,81 @@
+import dataclasses
+from typing import Any
+
+
+@dataclasses.dataclass()
```

Comment:
You can import just dataclass to avoid importing whole dataclasses module
Also you can remove parentheses here